bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,126 @@
+            reverted_commit_added_files_list = []
+            reverted_commit_deleted_files_list = []
+
+            for line in files_modified_by_reverted_commit:
+                pos = line.find('\t')
+                change = line[:pos]
+                if change == 'M':
+                    reverted_commit_modified_files_list.append(line[pos+1:])
+                elif change == 'A':
+                    reverted_commit_added_files_list.append(line[pos+1:])
+                elif change == 'D':
+                    reverted_commit_deleted_files_list.append(line[pos+1:])
+
+            for file_name in result.added_files:
+                if file_name not in reverted_commit_deleted_files_list:
+                    yield Result(self, 'Revert commit has an added '
+                                       'file %s that is not in reverted '
+                                       'commit.' % (file_name))
+
+            for file_name in result.deleted_files:
+                if file_name not in reverted_commit_added_files_list:
+                    yield Result(self, 'Revert commit has a deleted '
+                                       'file %s that is not in reverted '
+                                       'commit.' % (file_name))
+
+            for file_name in result.modified_files:
+                if file_name in reverted_commit_modified_files_list:
+                    head_commit_branch = run_shell_command(
+                        'git rev-parse --abbrev-ref HEAD')[0].strip('\n')
+
+                    get_revert_commit_diff_command = ('git diff HEAD^ HEAD ' +
+                                                      file_name)
+
+                    revert_commit_diff = run_shell_command(
+                        get_revert_commit_diff_command)[0]
+
+                    create_new_branch_command = ('git checkout -b ' +
+                                                 'inspectrevertbranch')
+                    run_shell_command(create_new_branch_command)
+
+                    create_expected_revert_commit = ('git revert %s --no-'
+                                                     'edit' %
+                                                     reverted_commit_sha)
+                    expected_revert_commit, err = run_shell_command(
+                        create_expected_revert_commit)
+                    if err:
+                        self.warn('Cannot compare the modified files.')
+                        run_shell_command('git revert --abort')
+                        return
+
+                    get_expected_commit_diff_command = ('git diff HEAD^ '
+                                                        'HEAD ' + file_name)
+                    expected_revert_commit_diff = run_shell_command(
+                        get_expected_commit_diff_command)[0]
+
+                    matcher = SequenceMatcher(
+                        None, revert_commit_diff, expected_revert_commit_diff)
```

Comment:
I think we also want to use `isjunk` parameter here (see the documentation of `difflib.SequenceMatcher`).
We want to mark the line that are not starting with `+` and `-` sign as junk, since we just care about the changes